recipes/lz4/all/conanfile.py

```diff
@@ -15,7 +15,7 @@ class LZ4Conan(ConanFile):
     license = ("BSD-2-Clause", "BSD-3-Clause")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/lz4/lz4"
-    topics = ("lz4", "compression")
```

Comment:
I would not remove lz4 as a topic, lz4 is not only library name but also compression algorithm name.

Reply:
the topics are for searching and the name automatically is included in that AFAIK so it should be fine